specify/specify7
# Es localization

Improve Spanish Localization (by `Iñigo Granzow de la Cerda <igranzow@rjb.csic.es>`)

To test:
review the changed files briefly to make sure nothing looks odd (i.e, make sure no stray `)` or `'`' were introduced)
